<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1021</width>
     <height>571</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
border-image: url(:/newPrefix/Untitled-1-01.jpg);
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>741</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Real-Time Video Surveillance &amp; Anomaly Detection</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Welcome To the Safer World</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>400</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(255, 255, 255);
border-image:none;
border-radius:20px;
	color: rgb(0, 0, 0);
}

QPushButton::hover{
	background-color: rgb(57, 114, 171);
color:rgba(255,255,255,230);
}
QPushButton::pressed{
	background-color: rgb(0, 182, 0);
color:rgba(255,255,255,230);
}</string>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>480</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(255, 255, 255);
border-image:none;
border-radius:20px;
	color: rgb(0, 0, 0);
}

QPushButton::hover{
	background-color: rgb(57, 114, 171);
color:rgba(255,255,255,230);
}
QPushButton::pressed{
	background-color: rgb(0, 182, 0);
color:rgba(255,255,255,230);
}</string>
    </property>
    <property name="text">
     <string>About</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>30</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;
image: url(:/newPrefix/Asset 3new.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>320</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(255, 255, 255);
border-image:none;
border-radius:20px;
	color: rgb(0, 0, 0);
}

QPushButton::hover{
	background-color: rgb(57, 114, 171);
color:rgba(255,255,255,230);
}

QPushButton::pressed{
	background-color: rgb(0, 182, 0);
color:rgba(255,255,255,230);
}</string>
    </property>
    <property name="text">
     <string>Test</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>480</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(255, 255, 255);
border-image:none;
border-radius:20px;
	color: rgb(0, 0, 0);
}

QPushButton::hover{
	background-color: rgb(57, 114, 171);
color:rgba(255,255,255,230);
}

QPushButton::pressed{
	background-color: rgb(0, 182, 0);
color:rgba(255,255,255,230);
}</string>
    </property>
    <property name="text">
     <string>Realtime Test</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
